recipe/meta.yaml

```diff
@@ -26,31 +32,34 @@ build:
 requirements:
   build:
     - python  # [win]
+    - 7za  # [win]
+    - curl  # [win]
+    - msinttypes  # [win and py27]
     - perl 5.22.2.1
-    - freetype 2.7|2.7.*
-    - fontconfig 2.12.*
+    - freetype 2.7|2.7.*  # [unix]
+    - fontconfig 2.12.*  # [linux]
     - openssl 1.0.*
     - jpeg 9*
     - zlib 1.2.*
     - libpng >=1.6.28,<1.7
-    - gst-plugins-base
+    - gst-plugins-base  # [linux]
     - icu 58.*
-    - libxcb
-    - dbus
+    - libxcb  # [linux]
```

Comment:
Given we have this on all platforms, we probably could drop the selector, but am not sure if there is a benefit in doing it. Maybe someone else can weigh in.

cc @pkgw